config.json

```diff
@@ -6,854 +6,1157 @@
   "solution_pattern": "[Ee]xample|\\.meta/solutions/[^/]*\\.rb",
   "exercises": [
     {
-      "slug": "hello-world",
-      "uuid": "4fe19484-4414-471b-a106-73c776c61388",
+      "slug": "gigasecond",
```

Comment:
Exercise order within this array used to be significant in V1.
Unless this has changed in V2 we probably need to keep the exercise ordering relatively consistent.

Reply:
I think the only things that should change are the
`core` and `unlocked_by` tags (and `auto-approve` on hello world)

Reply:
Does that mean I can merge #819 without causing you much trouble?